src/socket.cpp

```diff
@@ -365,6 +365,8 @@ void CSocket::OnDataReceived()
         use the signal/slot mechanism (i.e. we use messages for that).
     */
 
+    CHostAddress RecHostAddr;
```

Comment:
If we're moving this to a local variable, can we rename it to `haReceivedFrom`?  (Local variables shouldn't have leading capitals...  I know this is just to cut down on the changes, so it's non-blocking.)